src/AbstractEvent.php

```diff
@@ -19,7 +19,7 @@
     protected $emitter;
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
```

Comment:
@GrahamCampbell it inherrits everything like `@param` and `@return`, I've verified this with the phpDocumentor lead. What cases do you see where this altering something it shouldn't?